submissions/Ant-C-tech/tiny-js-world/index.js

```diff
@@ -0,0 +1,97 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/Ant-C-tech/a-tiny-JS-world
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+const dog = {
+   species: 'dog',
+   name: 'Spike',
+   gender: 'male',
+   legs: 4,
+   hands: 0,
+   saying: 'Woof-Woof!',
+   friends: ['John'],
+};
+
+const cat = {
+   species: 'cat',
+   name: 'Tom',
+   gender: 'male',
+   legs: 4,
+   hands: 0,
+   saying: 'Meow-Meow!',
+   friends: ['John'],
+};
+
+const woman = {
+   species: 'human',
+   name: 'Jane',
+   gender: 'female',
+   legs: 2,
+   hands: 2,
+   saying: 'Nice to meet you!',
+};
+
+const man = {
+   friends: ['Spike', 'Tom'],
+   species: 'human',
+   name: 'John',
+   legs: 2,
+   hands: 2,
+   saying: 'Hi, how are you?',
+   gender: 'male',
+};
+
+function createGreetingContent(obj) {
+
+   const propPrefixes = {
+      species: 'I am a %. ',
+      name: 'My name is %. ',
+      gender: 'My gender is %. ',
+      hands: 'I have % hands. ',
+      legs: 'I have % legs. ',
+      saying: 'I want to say you "%". ',
+      friends: 'I have friend %. ',
+   };
```

Comment:
Please avoid copy-pasting code examples. They might be not good enough and often intentinally are not good enough to give a student freedom to elaborate on this rather than just copy-paste.

In particular here variable name is bad. These are not prefixes but rather... erm... something else.

Reply:
Thank you for your review and for the opportunity to ask questions! What do you think, can I name this variable "template"?

Reply:
Thank you for your review! It's my mistake. What do you think, can I name the variable "template"?